diagnostic_feedback/config.py

```diff
@@ -0,0 +1,50 @@
+student_assets = {
+    'css': [
+        ('public/css/jquery.steps.css', 0),
+        ('public/css/message.css', 1),
+        ('public/css/student_style.css', 2)
+    ],
+    'js': [
+        ('public/js/libs/jquery.steps.js', 0),
+        ('public/js/src/common.js', 1),
+        ('public/js/src/student/student_quiz.js', 2)
+    ],
+    'js_classes': [
+        ('Common', 0),
+        ('StudentQuiz', 1)
+    ],
+    'templates': [
+        ('templates/student/quiz.html', 0),
+    ]
+
+}
+
+studio_assets = {
+    'css': [
+        ('public/css/jquery.steps.css', 0),
+        ('public/css/tooltip.css', 0),
```

Comment:
Ambiguous sort order between jquery.steps.css and tooltip.css.  Is this intentional?